cmd/apps/openfaas_app.go

```diff
@@ -60,33 +60,104 @@ func MakeInstallOpenFaaS() *cobra.Command {
 
 	openfaas.Flags().String("license-file", "", "Path to OpenFaaS Pro license file")
 
-	openfaas.RunE = func(command *cobra.Command, args []string) error {
-		appOpts := types.DefaultInstallOptions()
+	openfaas.PreRunE = func(cmd *cobra.Command, args []string) error {
```

Comment:
Looks much better.

Can we move the declaration under the RunE so that it's a bit less scrolling when we come in to fix/understand things later?

Reply:
Moved the declaration.